frontend/packages/kubevirt-plugin/locales/ja/kubevirt-plugin.json

```diff
@@ -0,0 +1,39 @@
+{
+  "Not available": "利用不可",
+  "Virtualization": "仮想化",
+  "Virtual Machines Templates": "仮想マシンテンプレート",
+  "{{name}} Details": "{{name}} の詳細",
+  "Network Interfaces": "ネットワークインターフェース",
+  "Disks": "ディスク",
+  "VM Template Details": "仮想マシンテンプレートの詳細",
+  "Scheduling and resources requirements": "スケジューリングおよびリソース要件",
+  "Virtual Machines": "仮想マシン",
+  "Overview": "概要",
+  "Details": "詳細",
+  "Console": "コンソール",
+  "Environment": "環境",
+  "Snapshots": "スナップショット",
+  "Operating system mismatch": "オペレーティングシステムの不一致",
+  "The operating system defined for this virtual machine does not match what is being reported by the Guest Agent. In order to correct this, you need to re-create the virtual machine with the correct VM selection. The disks of this virtual machine can be attached to the newly created one.": "この仮想マシンに定義されたオペレーティングシステムは、ゲストエージェントが報告する内容と一致しません。これを修正するには、正しい仮想マシンを選択して仮想マシンを再作成する必要があります。この仮想マシンのディスクは、新たに作成される仮想マシンに割り当てることができます。",
+  "Services": "サービス",
+  "Active Users": "アクティブユーザー",
+  "View Pending Changes": "保留中の変更の表示",
+  "Description": "説明",
+  "Operating System": "Operating System",
+  "Template": "Template",
```

Comment:
1. If "Operating System" could be translated in Japanese?
2. "Template" is not translated in both ja/kubevirt-plugin.json and zh/kubevirt-plugin.json.

Reply:
These look all set.